src/services/task1.ts

```diff
@@ -1,18 +1,22 @@
 import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
 import { IFile, IFileFormBody } from "../types";
 
+const BASE_URL = "http://localhost:8080";
+
 export const task1Api = createApi({
   reducerPath: "task1Api",
   baseQuery: fetchBaseQuery({
-    baseUrl: "http://localhost:8080",
+    baseUrl: `${BASE_URL}`,
   }),
+  tagTypes: ["Files"],
   endpoints: (builder) => ({
     getFiles: builder.query<string[], void>({
       query: () => "/api/v1/files",
       transformResponse: ({ files }) => files,
     }),
     getFileByName: builder.query<IFile, string>({
       query: (filename) => `/api/v1/files/${filename}`,
+      providesTags: ["Files"],
```

Comment:
Check this example: https://redux-toolkit.js.org/rtk-query/usage/automated-refetching#providing-cache-data
Use filename as id